frontend/src/components/app-input/app-input.styles.ts

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+
+export const InputWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+
+  label {
+    font-weight: bold;
+    margin: 0 0 5px 5px;
```

Comment:
Looks like this 5px should be variable for design consistency. This should be applied in more places